packages-web/datagrid/src/components/InfiniteBody.tsx

```diff
@@ -0,0 +1,49 @@
+import { createElement, Dispatch, ReactElement, ReactNode, SetStateAction, useCallback, useState } from "react";
+import classNames from "classnames";
+
+interface InfiniteBodyProps {
+    children: ReactNode;
+    hasMoreItems: boolean;
+    setPage?: Dispatch<SetStateAction<number>>;
```

Comment:
As we've seen `setPage` is actually of another type (e.g. it cannot handle a number). Shall we make it explicit in typings?

Reply:
Yes, will do that

Reply:
Done